docs/product-user-guide/src/main/asciidoc/decision-tables-rule-table-entries-ref.adoc

```diff
@@ -0,0 +1,87 @@
+[id='decision-tables-rule-table-entries-ref']
+= RuleTable definitions
+
+Entries in the `RuleTable` area of a decision table define conditions, actions, and other rule attributes for only the rules in that rule table.
+
+The following table lists the supported labels (column headers) and values for `RuleTable` definitions. For column headers, you can use either the given labels or any custom labels that begin with the letters listed in the table.
+
+*<@Michael (4 of 6): I've done my best to make the `Initial` column (now custom label column) more intuitive, but I think this feature is pretty weird and might do more harm that help. Is there a good reason why it's worth documenting it, rather than just ditching it and having users simply use the standard labels? Both here and in the attributes table, but oddly not in the RulSet table. It's not consistent and seems unhelpful. Thoughts?>*
```

Comment:
@sterobin I cannot explain the history of why we support custom codes beginning with the initial character and I've never seen it used to be honest.. however I guess documentation should reflect what is possible and not question whether the feature makes sense. This predates my arrival at Red Hat.

Reply:
I guess I disagree with you a bit. :) Some of the greatest product improvements, bug fixes, etc., spawn from the documentation process. Documenting a product reveals is weaknesses and ambiguities, before they get to the user. I'll think more on it and see if it's better to leave it or remove it. I was just curious how used it is. Thanks!

Reply:
@sterobin It was not my intention to cause offence. If I have, I apologise.

Reply:
Oh geeze, now I feel bad that I came off that way. Not at all! Sorry if I made it sound that way.